src/apps/btc/btc.c

```diff
@@ -97,15 +98,15 @@ bool app_btc_address_simple(
             script_type, keypath, keypath_len, params->bip44_coin)) {
         return false;
     }
-    struct ext_key derived_xpub __attribute__((__cleanup__(keystore_zero_xkey))) = {0};
```

Comment:
Been wondering... where's this ext_key coming from. Hm... no wally lib include. Ok, let's look at btc.h - I remember hearing something about 1:1 correspondence between .h and .c files - heh, not in there either. I wonder where it was coming from now :thinking: keystore.h? no... btc_params.h? no... aha, found an include in btc_common.h. That was so obivous, I hope I'll remember next time :)

Reply:
Not to mention of course that btc.c gets included wally lib headers via btc_common.h even though it doesn't seem to need it at all now.

Reply:
Since it is in the keystore interface, "obviously" the include you need is keystore.h to use it :joy: In keystore.h you have `#include <wally_bip32.h>` where you can find the definition.